ompi/mca/pml/ob1/pml_ob1_rdmafrag.h

```diff
@@ -46,7 +46,8 @@ struct mca_pml_ob1_rdma_frag_t {
     mca_bml_base_btl_t *rdma_bml;
     mca_pml_ob1_hdr_t rdma_hdr;
     mca_pml_ob1_rdma_state_t rdma_state;
-    size_t rdma_length;
+    size_t rdma_length;  /* how much the fragment will transfer */
+    opal_atomic_size_t rdma_bytes_remaining;  /* how much has been currently transferred */
```

Comment:
The comment is confusing. This variable holds remaining bytes.

Reply:
fixed.